tests/unit/converters/test_pandapower_converter_input.py

```diff
@@ -1565,8 +1572,13 @@ def test_get_trafo3w_winding_types__vector_group(mock_get_winding: MagicMock):
 
     expected = pd.DataFrame([[2, 1, 3], [1, 2, 0], [2, 1, 0]], columns=["winding_1", "winding_2", "winding_3"])
 
+    converter_empty = PandaPowerConverter()
```

Comment:
It would be cleaner to create a new unit test for this. `test_get_trafo3w_winding_types__vector_group__missing`